examples/official-storybook/stories/addon-docs/props.stories.mdx

```diff
@@ -1,9 +1,30 @@
 import { Meta, Props } from '@storybook/addon-docs/blocks';
```

Comment:
Did you mean to include this file in this PR?

Reply:
OOPS